subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -380,6 +380,24 @@ Internally, `AbstractArchiveTask` prefers the new properties and methods like `g
 
 If your plugin/build only uses these types (and does not extend them), nothing has changed.
 
+==== Archives fail on duplicates
+
+Until now archives defaulted to the `INHERIT` duplicates strategy, allowing the same path to exist multiple times in an archive.
```

Comment:
I think we need to be more specific what `archives` means here. I think we should reference `archive tasks`, and maybe point to `Zip` and `Tar` specifically.

Reply:
The previous header already defines it:
> ==== Change in behavior for tasks extending `AbstractArchiveTask` or subtypes (`Zip`, `Jar`, `War`, `Ear`, `Tar`)

Reply:
```suggestion
Until now archive tasks defaulted to the `INCLUDE` duplicates strategy, allowing the same path to exist multiple times in an archive.
```

I don't think people read the previous header. Let's repeat the affected tasks again here. Or have one section (changes for archive tasks) with two sub-sections.

Reply:
Done